### DISCUSSION TITLE
Application/Account naming vs Global/Local

### DISCUSSION THREAD

**Discussion by barnjamin** [algorand-devrel/beaker#108] [Polls]
This project chooses to call Global contract storage as Application state. The same is true for Local as Account State.

1) Is this a better name?
2) Is it worth it to depart from protocol terms for a better name?

**Comment by FrankSzendzielarz** [comment]
I would go further and propose the protocol update its terminology to match. Global and local are confusing terms.  The only potential issues I can see are a) from rhe Account perspective it's Application state, i.e. an account has many Application states, b) because the Account State is actually AccountApplixationRelationship state is it conceivable that future protocol changes would permit multiple OptIns for the same Account Application pair but with different arguments? Eg if Account was a person and Application was a bank and local state represented bank balance would 'person' be permitted to have multiple balances at 'bank'?  In this case is it always ok to call it Account state? Or is it Account Application State?

**Comment by iskysun96** [comment]
I definitely prefer Application / Account state but having two different names for the same thing will cause confusion. It would be great to deprecate global / local state and apply application / account state for everything, not just Beaker.
